src/pybind/mgr/dashboard/services/cephfs.py

```diff
@@ -260,3 +260,42 @@ def set_quotas(self, path, max_bytes=None, max_files=None):
         if max_files is not None:
             self.cfs.setxattr(path, 'ceph.quota.max_files',
                               str(max_files).encode(), 0)
+
+    def write_to_file(self, path, buf) -> None:
+        """
+        Write some data to the specified path.
+        :param path: The path of the file to write.
+        :type path: str.
+        :param buf: The str to write to the buf.
+        :type buf: str.
+        """
+        try:
+            fd = self.cfs.open(path, 'w', 644)
+            self.cfs.write(fd, buf.encode('utf-8'), 0)
+        except cephfs.Error as e:
+            logger.debug("EIO: %s", str(e))
+        finally:
+            self.cfs.close(fd)
+
+    def statfs(self, path) -> dict:
+        """
+        Get the statfs of the specified path by xattr.
+        :param path: The path of the directory/file.
+        :type path: str
+        :return: Returns a dictionary containing 'bytes',
+            'files' and 'subdirs'.
+        :rtype: dict
+        """
+        try:
+            bytes = int(self.cfs.getxattr(path, 'ceph.dir.rbytes'))
+        except cephfs.NoData:
+            bytes = 0
+        try:
+            files = int(self.cfs.getxattr(path, 'ceph.dir.rfiles'))
+        except cephfs.NoData:
+            files = 0
+        try:
+            subdirs = int(self.cfs.getxattr(path, 'ceph.dir.rsubdirs'))
+        except cephfs.NoData:
+            subdirs = 0
```

Comment:
```suggestion
        bytes = 0
        files = 0
        subdirs = 0
        with suppress(cephfs.NoData):
            bytes = int(self.cfs.getxattr(path, 'ceph.dir.rbytes'))
            files = int(self.cfs.getxattr(path, 'ceph.dir.rfiles'))
            subdirs = int(self.cfs.getxattr(path, 'ceph.dir.rsubdirs'))
```

Reply:
import `suppress` from `contextlib` (`contextlib` import already exists in 6th line of this line)

Reply:
`bytes = 0` Warn: Redefining built-in 'bytes' (redefined-builtin), rename to rbytes?

Reply:
done